programs/cli_wallet/cli_wallet.1

```diff
@@ -0,0 +1,81 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
+.TH CLI_WALLET "1" "September 2019" "cli_wallet 2.0.171025-minor-fix-1-2549-g991bd756" "User Commands"
```

Comment:
Hm, the version is wrong. And not nice.
I would like to avoid having to update this version manually in each release, and I'd like to avoid requiring help2man for the build. Perhaps build this optionally if help2man is available? Or rename to `cli_wallet.1.in` and have `cmake` replace the version at build time?

Reply:
> I would like to avoid having to update this version manually in each release

I agree with this.